### PR TITLE
DOC: Add copyto example

### PR DESCRIPTION
**Pull request by dg3192**

### CODE REVIEW

File: numpy/core/multiarray.py [hunk 1]
```diff
@@ -1107,6 +1107,21 @@ def copyto(dst, src, casting=None, where=None):
         A boolean array which is broadcasted to match the dimensions
         of `dst`, and selects elements to copy from `src` to `dst`
         wherever it contains the value True.
+
+    Examples
+    --------
+    >>> A = np.array([4,5,6])
+    >>> B = [1,2,3]
+    >>> np.copyto(A,B)
+        A
```
Comment (bsipocz) [inline numpy/core/multiarray.py:1116]: You need to add the `>>>` to the beginning of each and every command lines, which tells the testing infrastructure which ones are the one to run.
Here and below:
```suggestion
    >>> np.copyto(A, B)
    >>> A
```

File: numpy/core/multiarray.py [hunk 2]
```diff
@@ -1107,6 +1107,21 @@ def copyto(dst, src, casting=None, where=None):
         A boolean array which is broadcasted to match the dimensions
         of `dst`, and selects elements to copy from `src` to `dst`
         wherever it contains the value True.
+
+    Examples
+    --------
+    >>> A = np.array([4,5,6])
+    >>> B = [1,2,3]
```
Comment (bsipocz) [inline numpy/core/multiarray.py:1114]: Please follow PEP-8 python formatting, in practice for this pull request means to add a space after each coma.
```suggestion
    >>> B = [1, 2, 3]
```

**Review by bsipocz — CHANGES_REQUESTED** [review]

**Comment by seberg** [comment]
@dg3192 do you have time to follow up on the comments above?  Applying the changes is good here, but the same changes need to be fixed in a few more places and Brigitta did not mark each one individually.

**Comment by dg3192** [comment]
@seberg sure, I updated the example but I am having issues in pushing it back. Could you please help me on how should I do it?  
Screen Shot 2022-09-19 at 11 39 07 AM

**Comment by seberg** [comment]
@dg3192 since you did a force-push, I assume you figured it out?  There are still a few whitespace issues (e.g. space after comma missing).  Maybe you can try that out?

**Comment by dg3192** [comment]
@seberg I saved the changes that you highlighted but somehow I am getting error while pushing the changes. Not sure what should be the next step? Attaching error screenshot for your reference
Screen Shot 2022-09-21 at 2 03 58 PM

**Comment by seberg** [comment]
Maybe you worked on a different computer/setup?  The error says that what is on github appears to be newer than what you have locally.  That can happen when others push, but nobody else seemed to have pushed).

There could be other reasons though.  In any case, if you are sure that what you have locally is correct, simply add a `-f`, `git push -f`.  That is a "forced" push which will throw away any changes on github that are not also on your computer already.

**Comment by dg3192** [comment]
Ahh, my laptop got restarted yesterday and I think that's why this issue. I tried forced push as well but this is what I am getting.
Screen Shot 2022-09-21 at 2 49 17 PM

**Comment by seberg** [comment]
Try combining the `--set-upstream` with the `-f` :).

**Comment by dg3192** [comment]
@seberg I've forced push the changes. Is there anything else I should do from my end?

File: numpy/core/multiarray.py [hunk 3]
```diff
@@ -1107,6 +1107,22 @@ def copyto(dst, src, casting=None, where=None):
...
+    >>> np.copyto(A, B)
+    >>> A
+    array([1, 2, 3])
+
+    >>> A = np.array([[1, 2, 3], [4, 5, 6]])
+    >>> B = [[4, 5, 6], [7, 8, 9]]
+    >>> np.copyto(A, B)
+    >>> A
+    array([[4, 5, 6],
+       [7, 8, 9]])
```
Comment (seberg) [inline numpy/core/multiarray.py:1124]:
```suggestion
           [7, 8, 9]])
```

**Comment by dg3192** [comment]
Thank you!

**Comment by InessaPawson** [comment]
Hi-five on merging your first pull request to NumPy, @dg3192! We hope you stick around! Your choices aren’t limited to programming – you can review pull requests, help us stay on top of new and old  issues, develop educational material, work on our website, add or improve graphic design, create marketing materials, translate website content, write grant proposals, and help with other fundraising initiatives. For more info, check out: https://numpy.org/contribute
Also, consider joining our mailing list. This is a great way to connect with other cool people in our community and be part of important conversations that affect the development of NumPy: https://mail.python.org/mailman/listinfo/numpy-discussion

**Pull request merged**